website/docs/developer/deployment-guide/creating-orca-archive-bucket.md

```diff
@@ -154,6 +154,8 @@ modifications, which will be detailed below.
          },
          "Action":[
             "s3:GetObject*",
+            "S3:GetObjectTagging",
+            "S3:PutObjectTagging",
```

Comment:
Any specific reason to add these permissions?

Reply:
This is due to an issue uncovered by our GesDISC DAAC customer. When they created an *.cmr.xml file, it was tagged with the granule ID. These permissions allow use to copy the tag with the object. Since we do an exaxt copy, this was failing before we added these permissions. See the following article on tagging permissions https://medium.com/collaborne-engineering/s3-copyobject-access-denied-5f7a6fe0393e 

Jenny Liu put us on this path for the error. See slack thread https://eosdis.slack.com/archives/C76M787MZ/p1692826123858139